src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -60,10 +67,29 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
             var options = context.Options;
             var cancellationToken = context.CancellationToken;
 
+            var (token, semanticModel, symbols) = await GetSymbols(document, position, cancellationToken).ConfigureAwait(false);
+            var filteredSymbols = symbols.FilterToVisibleAndBrowsableSymbols(options.GetOption(CompletionOptions.HideAdvancedMembers, semanticModel.Language), semanticModel.Compilation);
```

Comment:
Moved into and propagated editorvisible option which is apparently not consumed by the VB side.